### ISSUE TITLE
Scheduler

### ISSUE THREAD

**Issue by off112a** [Sdahl1234/Sunseeker-lawn-mower#9]
Hi,

Is it possible to have multiple times in scheduler? For example one time in the morning and one in the afternoon.

**Comment by Sdahl1234** [comment]
No, It's not possible. But you can set the time from eg. 7-22 and add a automation to turn off the schedule at 12 and on again at 17.

**Comment by off112a** [comment]
Ok, thank you for the answer.
